browser-interface/scripts/publish.ts

```diff
@@ -22,6 +28,59 @@ async function checkFiles() {
   ensureFileExists(DIST_ROOT, "unity.loader.js")
 }
 
+async function getVersion(workingDirectory: string) {
+  const json = JSON.parse(readFileSync(workingDirectory + "/package.json", "utf8"));
+
+  let pkgJsonVersion = json.version;
+  if (!pkgJsonVersion) pkgJsonVersion = "0.0.0";
+
+  const version = semver.parse(pkgJsonVersion.trim());
+
+  if (!version) {
+    throw new Error("Unable to parse semver from " + pkgJsonVersion);
+  }
+
+  return `${version.major}.${version.minor}.${version.patch}`;
```

Comment:
This transformation is not necessary, the version _must_ be exactly the same as published. The publication updates the `package.json` so we are good returning `json.version`
```suggestion
  return json.version;
```